### DISCUSSION TITLE
Physician Fee Schedule - Modifiers, Facility vs Non Facility

### DISCUSSION THREAD

**Discussion by nsmartinez79** [CMSgov/price-transparency-guide#89] [Q&A] [answered]
There are potentially 6 possible fee schedule allowances for each procedure code on our physician fee schedule. Global, Professional, and Technical Components for services done in a facility setting and  Global, Professional, and Technical Components for services done in a non-facility (Office) setting. Global, Prof, and Technical are identified by modifiers. Facility or Non-Facility is determined by place of service codes. How are plans going to account for this with the current schema?

**Comment by shaselton-usds** [comment, marked as answer]
@nsmartinez79 Trying to wrap up old threads. 

The current schema allows for the distinction of [institutional vs professional](https://github.com/CMSgov/price-transparency-guide/blob/2df1618b6fe29f12cdcf55e81e5707da29c6608c/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L55) under [`billing_class`](https://github.com/CMSgov/price-transparency-guide/blob/2df1618b6fe29f12cdcf55e81e5707da29c6608c/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L55).

The schema also allows for [modifiers](https://github.com/CMSgov/price-transparency-guide/blob/2df1618b6fe29f12cdcf55e81e5707da29c6608c/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L56) to be included. Documentation can be found under the `billing_code_modifier` attribute in the [`Negotiated Price Object`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object)
